test/external-e2e/testdriver.yaml

```diff
@@ -3,6 +3,8 @@
 
 StorageClass:
   FromFile: /tmp/csi/storageclass.yaml
+SnapshotClass:
+  FromFile: /tmp/csi/snapshotclass.yaml
```

Comment:
pls also add `snapshotDataSource: true` in `Capabilities`

Reply:
it should be 
```
SnapshotClass:
    FromName: true
```

Reply:
> pls also add `snapshotDataSource: true` in `Capabilities`

I was hoping to add that in a followup PR but looking at the logs from `pull-csi-driver-nfs-sanity`, I guess it would be easier to include that here.